### ISSUE TITLE
Fontawesome icons in views are always black

### ISSUE THREAD

**Issue by thoellt** [ManiVaultStudio/core#140]
The icons on the view label tabs are always rendered in black, they should be rendered in the text color. Depending on the theme this is not always black (e.g in macOS dark-mode)

![image](https://user-images.githubusercontent.com/2159209/112138116-67c01780-8bd1-11eb-9d44-50ef55aaeed4.png)

**Comment by ThomasKroes** [comment]
I think this has been solved in the mean time. May I close this issue?

**Comment by thoellt** [comment]
I am not sure, did you do any fixes beyond those that I added? I only was able to fix a subset of these.

**Comment by ThomasKroes** [comment]
No, I did not. Perhaps I can move it to the release 1.0 project?

**Comment by ThomasKroes** [comment]
Fixed with the latest PR
